sysl panic when import content type "application/pdf"

Please **do not post any internal, closed source snippets** on this public issue tracker!

### Description
```yaml
openapi: "3.0.0"
info:
  version: 1.0.0
  title: Swagger Petstore
  license:
    name: MIT
servers:
  - url: http://petstore.swagger.io/v1
paths:
  /pets/{petId}/pdf:
    get:
      summary: PDF for a specific pet
      operationId: showPetById
      tags:
        - pets
      parameters:
        - name: petId
          in: path
          required: true
          description: The id of the pet to retrieve
          schema:
            type: string
      responses:
        '200':
          description: Expected response to a valid request
          content:
            application/pdf:
              schema:
                  type: string
                  format: binary
```

### Steps to Reproduce
1.  convert the yaml to sysl by command `sysl import -i .\codegen\testdata\simple\a.yaml -a test -o .\codegen\testdata\simple\a.sysl`

**Expected behavior:**
the yaml convert to sysl successful.

**Actual behavior:**
![image](https://user-images.githubusercontent.com/2755216/77249516-861ccb80-6c7c-11ea-9e58-b031021b03da.png)

* Debug&Crash Output
<!-- Full debug output can be obtained by running sysl with the flag `-v`. -->

```
PS E:\go\sysl-go> sysl import -i .\codegen\testdata\simple\a.yaml -a test -o .\codegen\testdata\simple\a.sysl -v
time="2020-03-22T20:36:36+08:00" level=debug msg="Logging: {loglevel:debug verbose:true}"
time="2020-03-22T20:36:36+08:00" level=info msg="Using swagger importer\n"
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x18 pc=0xb0a210]

goroutine 1 [running]:
github.com/anz-bank/sysl/pkg/importer.getSyslTypeName(0x0, 0x0, 0x0, 0x8)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/pkg/importer/utils.go:61 +0x130
github.com/anz-bank/sysl/pkg/importer.buildPathString(0xc000383ea0, 0x11, 0xc00043b1a0, 0x1, 0x1, 0x4, 0xc00043b1a0)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/pkg/importer/writer.go:205 +0x113
github.com/anz-bank/sysl/pkg/importer.(*writer).writeEndpoint(0xc00001da38, 0x12ea283, 0x3, 0xc000383ea0, 0x11, 0x0, 0x0, 0xc000106120, 0xc0004f0b40, 0x1, ...)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/pkg/importer/writer.go:222 +0x294
github.com/anz-bank/sysl/pkg/importer.(*writer).Write(0xc00001da38, 0xc00003a0f0, 0x4, 0x0, 0x0, 0xc0002faba0, 0x25, 0x12ef369, 0x7, 0xc0006a62a0, ...)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/pkg/importer/writer.go:62 +0x267
github.com/anz-bank/sysl/pkg/importer.importOpenAPI(0xc00003a0f0, 0x4, 0x0, 0x0, 0xc0002faba0, 0x25, 0x12ef369, 0x7, 0xc000351380, 0xc0000d6000, ...)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/pkg/importer/openapi.go:43 +0x338
github.com/anz-bank/sysl/pkg/importer.LoadSwaggerText(0xc00003a0f0, 0x4, 0x0, 0x0, 0xc0002faba0, 0x25, 0x12ef369, 0x7, 0xc000386300, 0x2c2, ...)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/pkg/importer/swagger.go:21 +0x11c
main.(*importCmd).Execute(0xc00043a3c0, 0x12ed48f, 0x6, 0x0, 0x0, 0x0, 0x1850960, 0x24ce528, 0xc0000d6000, 0x0, ...)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/cmd/sysl/cmd_import.go:82 +0x2e4
main.(*cmdRunner).Run(0xc000521b30, 0x12ed48f, 0x6, 0x1850960, 0x24ce528, 0xc0000d6000, 0x0, 0x0)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/cmd/sysl/cmd_runner.go:51 +0x598
main.main3(0xc0000b2000, 0x9, 0x10, 0x1850960, 0x24ce528, 0xc0000d6000, 0x237c080, 0x0)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/cmd/sysl/sysl.go:43 +0x232
main.main2(0xc0000b2000, 0x9, 0x10, 0x1850960, 0x24ce528, 0xc0000d6000, 0x16cb6c8, 0xc0000440b8)
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/cmd/sysl/sysl.go:88 +0x7c
main.main()
        C:/Users/杰/go/pkg/mod/github.com/anz-bank/sysl@v0.14.0/cmd/sysl/sysl.go:100 +0x80
```

### Your Environments
<!---
Run `sysl info` and `sysl env` to show the sysl binary info and related envvars, and paste the result between the ``` marks below.
If you are not running the latest version of sysl, please try upgrading because your issue may have already been fixed.
-->
```sh
$ sysl info
Build:
  Version      : unspecified
  Git Commit   : unspecified
  Date         : unspecified
  Go Version   : unspecified
  OS           : unspecified
Runtime:
  GOOS/GOARCH  : windows/amd64
  Go Version   : go1.13.3
...

$ sysl env
SYSL_MODULES=""
SYSL_PLANTUML=""
```

--- a/pkg/importer/swagger.go
+++ b/pkg/importer/swagger.go
@@ -44,13 +44,13 @@ func convertToOpenapiv3(data []byte) (*openapi3.Swagger, string, error) {
 }
 
 // nolint:gochecknoglobals
-var swaggerFormats = []string{"int32", "int64", "float", "double", "date", "date-time", "byte"}
+var swaggerFormats = []string{"int32", "int64", "float", "double", "date", "date-time", "byte", "binary"}
 
 func mapSwaggerTypeAndFormatToType(typeName, format string, logger *logrus.Logger) string {
 	typeName = strings.ToLower(typeName)
 	format = strings.ToLower(format)
 	if format != "" && !contains(format, swaggerFormats) {
-		logger.Errorf("unknown format '%s' being used, ignoring...\n", format)
+		logger.Warnf("unknown format '%s' being used, ignoring...\n", format)
 		format = ""
 	}
 
@@ -60,6 +60,7 @@ func mapSwaggerTypeAndFormatToType(typeName, format string, logger *logrus.Logge
 			"date":      "date",
 			"date-time": "datetime",
 			"byte":      StringTypeName,
+			"binary":    "bytes",
 		},
 		"integer": {
 			"":      "int",

--- a/pkg/importer/tests-openapi/multiple-return-values.sysl
+++ b/pkg/importer/tests-openapi/multiple-return-values.sysl
@@ -28,5 +28,5 @@ testapp "Simple" [package="package_foo"]:
     !type _test_ok:
         SimpleObj <: SimpleObj [mediatype="application/json"]:
             @json_tag = "SimpleObj"
-        string_ <: string [mediatype="application/pdf"]:
-            @json_tag = "string"
+        bytes_ <: bytes [mediatype="application/pdf"]:
+            @json_tag = "bytes"

--- a/pkg/importer/types.go
+++ b/pkg/importer/types.go
@@ -124,7 +124,9 @@ func (t TypeList) Sort() {
 type FieldList []Field
 
 // nolint:gochecknoglobals
-var builtIns = []string{"int32", "int64", "int", "float", "string", "date", "bool", "decimal", "datetime", "xml"}
+var builtIns = []string{"int32", "int64", "int",
+	"float", "string", "date", "bool", "decimal",
+	"datetime", "xml", "bytes"}
 
 func IsBuiltIn(name string) bool {
 	for _, kw := range builtIns {

